nimble_build_system/cad/shelf.py

```diff
@@ -489,7 +493,7 @@ def generate_shelf_model(self):
                 base_diameter=7
             )
 
-        return builder.get_body()
+        return builder.get_body().cq()
```

Comment:
This causes build to fail and causes an error on my machine. What is the `cq()`. I assume this is an issue with the CadScript abstraction of CadQuery?

Reply:
@julianstirling What is the error?

I'm not sure why it won't work for you. I have installed both cadorchestrator and nimble using `pip install -e .`, but I did re-create the environment after the PR before the last one that was just merged. That method call converts the CadScript object to a CadQuery object so that it can be used in an assembly. Otherwise I'll get an error. I can move that `.cq()` call somewhere else, but it may just throw an error for you in the new location.

Reply:
Ok, I see now. I fixed it in the latest commit. The command line you provided should run for you now.